oam/install_scripts/pre-uninstall

```diff
@@ -37,64 +34,61 @@ done
-else
-	$SUDO rm -f /etc/exports > /dev/null 2>&1
+if [ $user != "root" ]; then
+	export COLUMNSTORE_INSTALL_DIR=$installdir
+	export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$installdir/lib:$installdir/mysql/lib
 fi
 
-export COLUMNSTORE_INSTALL_DIR=$installdir
-
-cloud=`$COLUMNSTORE_INSTALL_DIR/bin/getConfig Installation Cloud`
+cloud=`$installdir/bin/getConfig Installation Cloud`
 if [ $cloud = "amazon-ec2" ] || [ $cloud = "amazon-vpc" ]; then
 	if test -f /etc/fstab ; then
-		$SUDO sed -i '/Columnstore\/data/d' /etc/fstab > /dev/null 2>&1
+	    sed -i '/Columnstore\/data/d' /etc/fstab > /dev/null 2>&1
+	fi
+fi
+
+#remove profile file
+if [ $user != "root" ]; then
+	if [ -f $prefix/.bash_profile ]; then		
+		profileFile=$prefix/.bash_profile
+	elif [ -f $prefix/.profile ]; then		
+		profileFile=$prefix/.profile
+	else
+		profileFile=$prefix/.bashrc
+		touch $prefix/.bashrc
+	fi
+
+	if [ -f ${profileFile}_backup ] ; then
+	    mv -f ${profileFile}_backup $profileFile
 	fi
+else
+	rm -f /etc/profile.d/columnstoreAlias.sh
 fi
- 
+
 #remove log file directories
-#rm -rf /var/log/mariadb/columnstore > /dev/null 2>&1
-#rm -f $installdir/mysql/db/*.err > /dev/null 2>&1
-$SUDO rm -f /var/log/mariadb/columnstore/activeAlarms > /dev/null 2>&1
-$SUDO rm -f /var/log/mariadb/columnstore/*.log1 > /dev/null 2>&1
-rm -rf $installdir/mysql/db/columnstore_log_archive > /dev/null 2>&1
-$SUDO rm -f /etc/default/columnstore
+if [ $user == "root" ]; then
+	rm -f /var/log/mariadb/columnstore/activeAlarms > /dev/null 2>&1
+	rm -f /var/log/mariadb/columnstore/*.log1 > /dev/null 2>&1
+fi
 
-# remove Mariab Columnstore Log Rotate File abd Transaction Log File
-$SUDO rm -f /etc/logrotate.d/columnstore > /dev/null 2>&1
-$SUDO rm -f /etc/cron.d/transactionLog > /dev/null 2>&1
+rm -rf $installdir/mysql/db/columnstore_log_archive > /dev/null 2>&1\
 
 # delete Mariab Columnstore shared memory segments
 $installdir/bin/clearShm  > /dev/null 2>&1
 
-# delete prat files
-$SUDO rm -f /etc/cron.d/ps > /dev/null 2>&1
-$SUDO rm -f /etc/pscollect > /dev/null 2>&1
-$SUDO /etc/init.d/crond reload  > /dev/null 2>&1
-$SUDO systemctl reload crond.service > /dev/null 2>&1
-
-# delete tmp files
+#get temp directory
+tmpDir=`$installdir/bin/getConfig SystemConfig SystemTempFileDir`
+if [[ $tmpDir = *"tmp" ]]; then
+	# delete tmp files
+	rm -rf $tmpDir/*
+fi
 rm -f $installdir/local/*.columnstore
 rm -rf $installdir/local/etc/
-rm -rf /tmp/bucketreuse
-rm -f /tmp/columnstore.txt
-rm -f /tmp/dbbuilder.*
-rm -f /tmp/dbrmfiles
-$SUDO rm -f /var/lock/subsys/columnstore
-rm -f /tmp/pkgcheck
-rm -f /tmp/upgrade-status.log.*
-rm -f /tmp/mount.log
-rm -f $installdir/data/bulk/tmpjob/* >/dev/null 2>&1
-rm -rf /tmp/columnstore_tmp_files
+rm -f $installdir/data/bulk/tmp/job/* >/dev/null 2>&1
 rm -f $installdir/local/moveDbrootTransactionLog
 
-#delete any old setenv files under home directory
-if [ $user = "root" ]; then
-	rm -f /root/setenv-hdfs*
-else
-	rm -f /home/$user/setenv-hdfs*
```

Comment:
Is this the right place?  Are we putting scripts in the user's home dir?  If so, let's not do that.